src/dl_txt_pack.cpp

```diff
@@ -1111,6 +1113,33 @@ static void dl_txt_pack_eat_and_write_struct( dl_ctx_t dl_ctx, dl_txt_pack_ctx*
 	}
 }
 
+struct SSubInstance
+{
+	dl_substr name;
+	size_t pos;
+	uint32_t name_hash;
+};
+
+struct dl_internal_sort_pred
+{
+	__forceinline bool operator()(const SSubInstance& i1, const SSubInstance& i2)
```

Comment:
msvc only right? Is it enough with just an inline here? Otherwise we will have to add a DL_FORCEINLINE.

Reply:
Force inline means optimizing the debug build. I honestly haven't even measured this one, it was just an old habit that kicked in for this kind of code.
I have made a define for it in later versions, after I've added continuous integration for GCC (you got that in the latest PR).
It was needed for another use case when I've actually measured a >10%/many-second improvement in Debug by adding it.